src/ElvTenant.js

```diff
@@ -721,6 +721,50 @@ class ElvTenant {
     return res;
   }
 
+  async TenantGetFaucet({ asUrl, tenantId }) {
```

Comment:
did you intend to call this / made an accessor for it?

Reply:
added to tenant_show cmd